### PR TITLE
Fix/file browser

### PR DESCRIPTION
**Pull request by felixdausch**
@plentymarkets/team-terra
@plentymarkets/ceres-io 

#### Must be done by Terra

Testing
- [x] Person 1 (mandatory)
- [x] Person 2 (mandatory)

Information transfer
- [x] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Browser-Support (relevant for changes in scss / appearance of components and jQuery)
- [x] Chrome
- [x] Firefox
- [x] Safari

----

#### Must be done by every developer
Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline). 

Documentation
- [x] Changelog (optional: relevant for every change which influences the API)
- [ ] ~~Example (updated or created)~~
- [ ] ~~JSDoc (optional: relevant for every change which influences the API)~~

Testing
- [ ] ~~Unit Test (updated or created)~~

### CODE REVIEW

**Comment by dantrau** [comment]
Restart tests.

**Comment by MFrank84** [comment]
Bildschirmfoto 2019-11-01 um 16 13 09


Die Progressbar schwebt "in der Luft", wenn ich während des Uploadvorgangs scrolle

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +15,7 @@
 
 ### Bug Fixes
 * **radio-group** Adjusted height of `<legend>` and only is rendered when content is available.
+* **file-browser** Restrict number of parallel uploads
```
Comment (pweyrich) [inline CHANGELOG.md]:
```suggestion
```

**Comment by maxhelligeplenty** [comment]
Hallo  ,
MFrank84 approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1057#pullrequestreview-310500504

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Die Progressbar schwebt "in der Luft", wenn ich während des Uploadvorgangs scrolle -- You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1057#issuecomment-548826127
Test.test.test

**Pull request merged**